packages/webview_flutter/example/integration_test/webview_flutter_test.dart

```diff
@@ -1272,18 +1276,19 @@ void main() {
       ),
     );
     final WebViewController controller = await controllerCompleter.future;
-    await controller.evaluateJavascript('window.open("about:blank", "_blank")');
+    await controller
+        .evaluateJavascript('window.open("https://flutter.dev/", "_blank")');
     await pageLoaded.future;
     final String? currentUrl = await controller.currentUrl();
-    expect(currentUrl, 'about:blank');
+    expect(currentUrl, 'https://flutter.dev/');
   });
 
   testWidgets(
     'can open new window and go back',
```

Comment:
Fixed the weird bug where `pageLoaded.complete()` was called in the callback multiple times. From the Dart side, this caused a silent error and prevented the rest of the method calls from going through.